### PR TITLE
Release Notes: add information regarding experimental gsettings backend

### PR DESCRIPTION
**Pull request by sirblackheart**
add experimental GSettings notes to release notes for 0.8.17

### CODE REVIEW

File: doc/NEWS.md [hunk 1]
```diff
@@ -117,6 +118,31 @@ issues and setting up travis:
...
+As part of the ongoing work of the bachelor thesis
+`Integration of Elektra into the GNOME desktop environment`
+we now have experimental support for Elektra as a GSettings backend on Linux
+(We will look into getting OS X support on a later date).
+When installed, applications using GSettings default backend will write to Elektra
+below the `/sw` key. The GSettings bindings are intended as a preview version so
+please do not use them in a production system.
+
+To build the GSettings backend you have to explicitly add it to bindings.
+e.g. `-DBINDINGS=gsettings`
```
Comment (markus2330) [inline doc/NEWS.md]: it should be mentioned that ALL does not include it. Does "ALL;gsettings" work?

File: doc/NEWS.md [hunk 2]
```diff
@@ -117,6 +118,31 @@ issues and setting up travis:
...
+
+To manually install the backend, after building it copy the `libelektrasettings.so` into
+the gio module dir. Both `GElektra` and `Elektra` libraries have to be installed for the
```
Comment (markus2330) [inline doc/NEWS.md]: Does make install not work? I would not suggest to copy files from build directory somewhere else.

**Comment by markus2330** [comment]
Looks fine!

File: doc/NEWS.md (hunk 2 above)
Comment (sirblackheart) [inline doc/NEWS.md]: Make install should work fine. I will delete the line.

**Pull request merged**